lib/genesis/genesis_test.go

```diff
@@ -6,47 +6,59 @@ package genesis
 import (
 	"testing"
 
-	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/assert"
```

Comment:
Why assert over require here?

Reply:
I didn't even realize that got changed 😅
In my defense (but i'll revert it), `assert` is better when you just want to report the test error but there is no need to fail the test right now. So I use `require` really when test execution should stop (i.e. failed to create a test service).

Reply:
reverted